src/libsystemd-network/sd-radv.c

```diff
@@ -569,6 +580,42 @@ int sd_radv_set_preference(sd_radv *ra, unsigned preference) {
         return 0;
 }
 
+int sd_radv_set_home_agent_information(sd_radv *ra, int home_agent) {
+        assert_return(ra, -EINVAL);
+
+        if (ra->state != RADV_STATE_IDLE)
+                return -EBUSY;
+
+        SET_FLAG(ra->flags, ND_RA_FLAG_HOME_AGENT, home_agent);
+
+        return 0;
+}
+
+int sd_radv_set_home_agent_preference(sd_radv *ra, uint16_t preference) {
+        assert_return(ra, -EINVAL);
+        assert_return(preference < UINT16_MAX, -EINVAL);
```

Comment:
The man page said the setting accept `UINT16_MAX`. I think the assertion should be dropped.